### PR TITLE
Implemented Disclaimer

### PR DESCRIPTION
**Pull request by alessandrofavero**
Implemented disclaimer #47 

Onboarding disclaimer
onboarding_disclaimer

Agreed disclaimer
agreed_disclaimer

Font 'Legal' is not implemented, waiting for #79 , I will implement it before merging.

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

**Comment by lukesleeman** [comment]
@alessandrofavero you might want to make sure PR #79 gets merged?  Seems that the code in this screen would depend on the text style in #79 ?

**Comment by maks** [comment]
@alessandrofavero I think you just need to fix conflicts with latest master.

**Comment by alessandrofavero** [comment]
> @alessandrofavero you might want to make sure PR #79 gets merged? Seems that the code in this screen would depend on the text style in #79 ?

Correct, like I said in the PR description, I will merge it (with the right font) after #79 gets merged.

**Comment by alessandrofavero** [comment]
> @alessandrofavero I think you just need to fix conflicts with latest master.

ok @maks, I'm at work now but I will merge master into the branch later today

**Comment by alessandrofavero** [comment]
@lukesleeman @maks I've merged master and updated font to Legal

**Comment by alessandrofavero** [comment]
Problem with Legal font at 1920x1080.
Screenshot_1585204420

File: test/widget_test.dart [hunk 1]
```diff
@@ -12,7 +12,5 @@ void main() {
   testWidgets('show WH org name on home page', (tester) async {
     // dont use _initApp as for this we *dont* want the future to complete yet
     await tester.pumpWidget(MyApp());
```
Comment (maks) [inline test/widget_test.dart]: Just for posterity: I told @alessandrofavero  he could disable the test and I would fix up the test to work with his changes in subsequenet PR

**Review by maks — APPROVED** [review]
Just for posterity: I told @alessandrofavero he could disable the test and I would fix up the test to work with his changes in subsequent PR

**Pull request merged**